src/compiler/checker.ts

```diff
@@ -125,7 +125,9 @@ namespace ts {
         let anySignature = createSignature(undefined, undefined, emptyArray, anyType, undefined, 0, false, false);
         let unknownSignature = createSignature(undefined, undefined, emptyArray, unknownType, undefined, 0, false, false);
 
-        let globals: SymbolTable = {};
+        let globalScope: Scope = {

```

Comment:
I get why it's nice to have this, but you don't actually end up using `globalScope` on its own - we always just grab `symbols` anyhow.